pinot-common/src/main/java/org/apache/pinot/common/minion/MinionClient.java

```diff
@@ -72,8 +72,8 @@ public Map<String, String> scheduleMinionTasks(@Nullable String taskType, @Nulla
       throws IOException, HttpException {
     HttpPost httpPost = createHttpPostRequest(
         MinionRequestURLBuilder.baseUrl(_controllerUrl).forTaskSchedule(taskType, tableNameWithType));
-    HttpResponse response = HTTP_CLIENT.execute(httpPost);
-    int statusCode = response.getStatusLine().getStatusCode();
+    CloseableHttpResponse response = HTTP_CLIENT.execute(httpPost);
```

Comment:
Do we need to close the response (wrap it in try-with-resource block)? Same for other places

Reply:
Fixed MinionClient.java -- next, I will review other places

Reply:
I went through the entire pull request. We are now closing the CloseableHttpResponse instances.